CSharpCollections1/Citizen.cs

```diff
@@ -0,0 +1,43 @@
+using System.Runtime.InteropServices.JavaScript;
+using Microsoft.VisualBasic;
+
+namespace CSharpCollections1;
+
+public class Citizen
+{
+    private string _lastName;
```

Comment:
add spaces between all fields and properties and remove unnecessary. Check for all classes.